### PR TITLE
Refactor: Linter test for only class-sided default values

### LINKED ISSUE #360: Alle default Attribute einheitlich halten (Klasse / Instanz) + lintertest

### PR DESCRIPTION
**Pull request by jb3rndt**
Fixes #360

### CODE REVIEW

File: packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st [hunk 1]
```diff
@@ -1,4 +1,4 @@
-default values
+drawing
```
Comment (phkeese) [inline packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st:1]: is this not a default value?

**Review by phkeese — COMMENTED** [review]
looks good, maybe we could even try to assert for constant values in those default values. This does not provide a way to find default values not named `default*`, but that could be quite hard tom implement.

File: packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st (hunk 1 above)
Comment (jb3rndt) [reply packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st:1]: I wasnt sure there. `maxContentWidth` is contructed by other default values, so it did not hold an independent default value. Thus it somehow just behaves like a calculating method.

**Comment by jb3rndt** [comment]
> 
> 
> looks good, maybe we could even try to assert for constant values in those default values. This does not provide a way to find default values not named `default*`, but that could be quite hard tom implement.

Yep, I thought about that as well and was not sure how that would behave. Would a default value like `10 @ 10` be recognized as a constant? It would be bad if not. Additionally, look at `defaultTextMorph` for example. It returns a default morph which clearly is no constant. However, on could argue that this should not be classified as a default value.
To sum it up, I think looking for constants would maybe raise more problems than looking for `default...` already does.
But we could actually use looking for contants to detect methods that should be default methods. (Would be a different linter test then)

File: packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st (hunk 1 above)
Comment (phkeese) [reply packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st:1]: Fair enough

**Review by phkeese — APPROVED** [review]
Looks good. We can discuss further linter tests another time

**Review by phkeese — APPROVED** [review]
looks good

**Pull request merged**